tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -2389,3 +2389,308 @@ Feature: Send a sharing invitations
+    When user "Alice" tries to send the following share invitation using root endpoint of the Graph API:
+      | space           | NewSpace           |
+      | sharee          | Brian, Carol       |
+      | shareType       | user, user         |
+      | permissionsRole | <permissions-role> |
+    Then the HTTP status code should be "400"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["error"],
+        "properties": {
+          "error": {
+            "type": "object",
+            "required": [
+              "code",
+              "innererror",
+              "message"
+            ],
+            "properties": {
+              "code": {
+                "const": "invalidRequest"
+              },
+              "innererror": {
+                "type": "object",
+                "required": [
+                  "date",
+                  "request-id"
+                ]
+              },
+              "message": {
+                "const": "Key: 'DriveItemInvite.Recipients' Error:Field validation for 'Recipients' failed on the 'len' tag"
+              }
+            }
+          }
+        }
+      }
+      """
+    Examples:
+      | permissions-role |
+      | Space Viewer     |
+      | Space Editor     |
+      | Manager          |
+
+
+  Scenario Outline: try to send share invitation on personal drive to user with different roles using root endpoint
+    When user "Alice" tries to send the following share invitation using root endpoint of the Graph API:
+      | space           | Personal           |
+      | sharee          | Brian              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
+    Then the HTTP status code should be "400"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["error"],
+        "properties": {
+          "error": {
+            "type": "object",
+            "required": [
+              "code",
+              "innererror",
+              "message"
+            ],
+            "properties": {
+              "code": {
+                "const": "invalidRequest"
+              },
+              "innererror": {
+                "type": "object",
+                "required": [
+                  "date",
+                  "request-id"
+                ]
+              },
+              "message": {
+                "const": "unsupported space type"
+              }
+            }
+          }
+        }
+      }
+      """
+    Examples:
+      | permissions-role |
+      | Viewer           |
+      | File Editor      |
+      | Viewer           |
+      | Editor           |
+      | Uploader         |
+
+
+  Scenario Outline: try to send share invitation on shares drive to user with re-sharing permissions using root endpoint
```

Comment:
```suggestion
  Scenario Outline: try to send share invitation on shares drive to user with different permissions using root endpoint
```

Reply:
its re-sharing permission so its better if we specify that